### PR TITLE
Don't require requests in hacking/report.py

### PR DESCRIPTION
**Pull request by sivel**
##### SUMMARY
Don't require requests in hacking/report.py
##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
hacking/report.py

##### ANSIBLE VERSION

```paste below
2.8
```

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: hacking/report.py [hunk 1]
```diff
@@ -5,11 +5,13 @@
 from __future__ import (absolute_import, print_function)
 
 import argparse
+import json
 import os
-import requests
 import sqlite3
 import sys
 
+from ansible.module_utils.urls import open_url
```
Comment (mattclay) [inline hacking/report.py]: Move this import down next to the other ansible import. That way the script will continue to work if you haven't sourced hacking/env-setup yet.

**Review by mattclay — CHANGES_REQUESTED** [review]

File: hacking/report.py (hunk 1 above)
Comment (sivel) [reply hacking/report.py]: done

**Review by mattclay — APPROVED** [review]

**Review by webknjaz — COMMENTED** [review]
Why?

**Comment by sivel** [comment]
> Why?

I don't keep `requests` installed in my virtualenv as it is unnecessary fluff for my day to day dev needs.

I have to install it to run this script, then uninstall it and it's deps once I am done.  I keep my ansible dev environment restricted to the base deps.  Only installing things on an as needed basis.

**Pull request merged**